property/views.py

```diff
@@ -147,6 +146,29 @@ class CreateAndListPropertyView(generics.ListCreateAPIView):
     filter_class = PropertyFilter
     parser_classes = (MultiPartParser, FormParser)
 
+    @staticmethod
+    def validate_building_rooms(data):
```

Comment:
@wangonya, Nice try with if-else statement!

In Django to validate the data we basically use serializers, did you try them?
For your condition, you may check Truthy and Falsy concepts in python.
For more info check [this](https://stackoverflow.com/questions/39983695/what-is-truthy-and-falsy-how-is-it-different-from-true-and-false) and [this](https://medium.com/the-andela-way/idiomatic-python-coding-the-smart-way-cc560fa5f1d6)